### PR TITLE
[SEM-91] #91 : Component of volume management

### PR DESCRIPTION
**Pull request by 11royer**
We need to implement a component that contains the elements shown in the image, thus containing all the form fields, as well as integrating the editing component shown in the image.

### CODE REVIEW

**Comment by 11royer** [comment]
This is my ticket URL:
https://ditmar.github.io/backoffice-openscience/branch-SEM-91/?path=/story/ui-components-atoms-volume-management

File: src/ui/components/atoms/volume-management/VolumeManagement.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import classNames from 'classnames';
```
Comment (Ditmar) [inline src/ui/components/atoms/volume-management/VolumeManagement.tsx:1]: this component don't should be a atom, this component must be an organism

File: src/ui/components/atoms/volume-management/VolumeManagement.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  variant = 'primary',
+  size = 'medium',
+}: IVolumeManagementProps) {
+  return (
+    <div className={classNames(styles['volume-management'], styles[size])} role="region">
+      <div className={styles['image-container']}>
+        <p>Portrait</p>
+        <div className={styles['image-placeholder']}>
+          <Icon src={ICONS.image} className={styles['image-icon']} />
+          <span>Click to add an asset or drag and drop one in this area</span>
```
Comment (Ditmar) [inline src/ui/components/atoms/volume-management/VolumeManagement.tsx:57]: this should be a separate component, only for managing images, and it is necessary to add the drag events to added files there, this component should be a molecule,

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments

**Pull request closed without merging**